code/modules/spells/targeted/retard.dm

```diff
@@ -7,7 +7,7 @@
 	school = "evocation"
 	charge_max = 200 // 20 seconds
 	//Invocation is noted below
-	invocation_type = SpI_WHISPER //Wizard will whisper what they say instead of shouting
+	invocation_type = SpI_SHOUT //Wizard will whisper what they say instead of shouting
```

Comment:
this comment is a lie

Reply:
So is the cake but one is a popular culture reference and the other is a Portal meme